hummingbot/client/hummingbot_application.py

```diff
@@ -1215,4 +1295,3 @@ def bounty(self, register: bool = False, status: bool = False, terms: bool = Fal
         self.app.change_prompt(prompt=">>> ")
         self.app.toggle_hide_input()
         self.placeholder_mode = False
-
```

Comment:
don't delete empty line, standard practice is to leave empty line at end of a file